1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,22 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+
+
+
+
+newArray = arr.forEach(function (e) {
+  if(e % 3 == 0) {
+  console.log('Fizz');
+} else if(e % 5 == 0) {
+  console.log('Buzz')
+} else (e % 3 && e % 5 == 0)
```

Comment:
This isn't correct. Since the condition in line 22 will never happen, because the condition in either line 17 or line 19 will return true first